robottelo/vm.py

```diff
@@ -302,6 +304,8 @@ def register_contenthost(self, org, activation_key=None, lce=None,
                 username,
                 password,
             )
+            if auto_attach:
+                cmd += u' --auto-attach'
```

Comment:
Does auto attach work only with lce&credentials, won't it work with org+activation key?